providers/queuing/serviceBusFactory.js

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
```

Comment:
update these to be the same as the rest of GHCrawler

Reply:
As discussed, we'll leave it for now.